src/mon/FSCommands.cc

```diff
@@ -268,6 +287,37 @@ class FsNewHandler : public FileSystemCommandHandler
       mon->osdmon()->wait_for_writeable(op, new PaxosService::C_RetryMessage(mon->mdsmon(), op));
       return -EAGAIN;
     }
+
+    bool recover = false;
+    cmd_getval(cmdmap, "recover", recover);
+
+    auto&& fs = fsmap.create_filesystem(fs_name, metadata, data,
+        mon->get_quorum_con_features(), fscid, recover);
+
+    // set fs options
+    int ret = 0;
+    string set_fsops_info;
+    for (int i = 1 ; i < (int)fsops_vec.size() ; i+=2) {
```

Comment:
```suggestion
    for (size_t i = 1 ; i < fsops_vec.size() ; i+=2) {
```

Reply:
done, thanks!

Reply:
i think better would be use auto instead of specifying the variable?